src/protocols/bdx/AsyncTransferFacilitator.cpp

```diff
@@ -175,16 +187,15 @@ void AsyncResponder::NotifyEventHandled(const TransferSession::OutputEventType e
     // the transfer session to handle end-of-transfer (e.g. in some error conditions it might want to send
     // out a status report).
```

Comment:
This comment is not longer matching the code, right?

I think more importantly: we should document why we don't care about `status` for these event types.